### ISSUE TITLE
Is the JAR file missing as I couldn't see the mysql-connectot-java.jar do we need one to connect to mysql database

### ISSUE THREAD

**Issue by adetheprince** [shivamvk/SocailNetworkAndroidApp#10]
I am totally new and I am a bit confused about the JAR file that every tutorial mentioned.
I can see you have a java class Constants that you inserted link to jsp files.
I have uploaded all these file to my ubuntu server where mysql installed, I have changed the mysql address from "jdbc:mysql://localhost/facebookandroid?user=root&password=test123" to mine example "jdbc:mysql://localhost/facebookandroid?user=root&password=mypassword" on every jsp but I am not able to connect or insert any data to my database, I have tested the register page to no availe, not sure where I am wrong, I am also sure the database called facebookandroid does exist and mysql is started and running, I also have user called root with password mypassword, after tested the file with android studio I get the actual jsp file code printed out on the screen as its an error toast , strange that I cant see any error in AS log, here is the image can you please help me or guide me where I am going wrong.

`<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
    pageEncoding="ISO-8859-1" import="java.sql.*"%>
    <%@page import="org.json.*" %>

<%
Class.forName("com.mysql.jdbc.Driver");
Connection con = DriverManager.getConnection("jdbc:mysql://localhost/facebookandroid?user=root&password=adilimaam");
PreparedStatement statement = con.prepareStatement("select * from users where userEmail=?");
statement.setString(1, request.getParameter("userEmail"));
ResultSet rs = statement.executeQuery();
if(rs.next()){
	response.getWriter().append("error");
} else {
	PreparedStatement statement1 = con.prepareStatement("insert into users(userName,userEmail,userPassword) values(?,?,?)");
	statement1.setString(1, request.getParameter("userName"));
	statement1.setString(2, request.getParameter("userEmail"));
	statement1.setString(3, request.getParameter("userPassword"));
	statement1.executeUpdate();
	response.getWriter().append("okay");
}
con.close();
%>

<%!
public static void convertResultSetIntoJSON(ResultSet resultSet, HttpServletResponse response) throws Exception {
    JSONArray jsonArray = new JSONArray();
    while (resultSet.next()) {
        int total_rows = resultSet.getMetaData().getColumnCount();
        JSONObject obj = new JSONObject();
        for (int i = 0; i < total_rows; i++) {
            String columnName = resultSet.getMetaData().getColumnLabel(i + 1).toLowerCase();
            Object columnValue = resultSet.getObject(i + 1);
            // if value in DB is null, then we set it to default value
            if (columnValue == null){
                columnValue = "null";
            }
            /*
            Next if block is a hack. In case when in db we have values like price and price1 there's a bug in jdbc - 
            both this names are getting stored as price in ResulSet. Therefore when we store second column value,
            we overwrite original value of price. To avoid that, i simply add 1 to be consistent with DB.
             */
            if (obj.has(columnName)){
                columnName += "1";
            }
            obj.put(columnName, columnValue);
        }
        jsonArray.put(obj);
    }
    response.getWriter().write(jsonArray.toString());
}
%>
`

**Comment by shivamvk** [comment]
Yes you need to add a mysql connector jar. And can you please the logs so that i can tell you exactly why it's not working!

**Comment by adetheprince** [comment]
Thanks for the quick reply body, as I said there are no log error in android studio but the app shows all these text on the screen of the phone as Toast when I try to register or even login, its just strange that I see the whole Register.jsp is fetched on the screen like an error but its not any error log its just the code is fetched instead of reading from mysql and showing result. can you tell me where to upload the JAR file as I dont see libs folder here?

**Comment by adetheprince** [comment]
![image](https://user-images.githubusercontent.com/15795755/85030577-82aeb880-b175-11ea-98d2-b105d5ee08fc.png)

**Comment by adetheprince** [comment]
![image](https://user-images.githubusercontent.com/15795755/85030640-9a863c80-b175-11ea-9191-d4a0809b24eb.png)

this is on the register activity after i tried to register a test user. I am sure my database mysql is not connecting due to the JAR file is missing as i am not sure how to do that in android studio since there is no libs folder or can you share the jar file for test purpose please .
thanks

**Comment by adetheprince** [comment]
Any help on why your code fetched on the screen like that instead of opening normal, I am sure there are something wrong with the code as my mysql is working fine.
 Thanks,
